text.py

```diff
@@ -4,46 +4,28 @@
 import os
 from PIL import Image, ImageDraw, ImageFont
 
-FONTINDEX = 0
+def validate_font(fontpath: str) -> bool:
+    """Validate if the font exists
 
-def validate_font(fontname: str) -> bool:
-    """Validate if the font exists in the font directory
-    
     Args:
-        fontname (str): Path to the font file
-    
+        fontpath (str): Path to the font file
+
     Returns:
         bool: True if font exists, otherwise False
     """
-    if not os.path.isfile(fontname):
-        print(f"Error: Font '{fontname}' not found in the font directory.")
-        return False
-    return True
+    return os.path.isfile(fontpath)
 
-def create_font(size: int, fontname: str) -> ImageFont.FreeTypeFont:
+def create_font(size: int, fontpath: str) -> ImageFont.FreeTypeFont:
     """Create the font object
 
     Args:
         size (int): Pixel size
-        fontname (str): Path to the font file
+        fontpath(str): Path to the font file
 
     Returns:
         ImageFont.FreeTypeFont: The created font
     """
-    font = ImageFont.truetype(fontname, size)
-    return font
-
-def create_bold_font(size: int, fontname: str) -> ImageFont.FreeTypeFont:
```

Comment:
I believe the removal of this is because we use the `create_font` function for both regular & bold fonts now?

Does this work if `font` and **`bold font`** are different styles?

Reply:
That is  a good point. I'll close this PR now as it is getting quite large and covering a few bases, then how about we create another PR to add support for a font index parameter which defaults to None or 0 (need to test), and gets passed through to this create_font function?